releases/v19.2.0-beta.20190930.md

```diff
@@ -11,8 +11,8 @@ In addition to bug fixes and various general, enterprise, SQL, and command-line
 - **Locality-aware backup and restore**: You can now back up or restore a subset of a table's data according to [zone configurations](../v19.2/configure-replication-zones.html) (e.g., send your EU backup partition to an EU cloud storage location).
 - [**Import performance improvements**](../v19.2/import.html): Internal improvements have been made to `IMPORT` to enable more efficient loading of large data sets.
 - **Show partitions using SQL**: CockroachDB now supports the [`SHOW PARTITIONS`](../v19.2/show-partitions.html) statement, which can be used to identify partitions. It also enhances [`SHOW CREATE TABLE`](../v19.2/show-create.html) to show constraints as described.
-- **Productionize Vectorization Prototype**: Queries are now automatically faster for users. This work expands upon the SQL Vectorization Prototype and allows it to be used in production. Note, this does not cover the long-tail of features necessary to use vectorization on all queries. Vectorized execution is turned on by default for a subset of queries and default back to DistSQL execution for unsupported queries
-- **Improve Streaming Behavior on JDBC Driver**: PostgreSQL wire protocol cursors have been implemented. As a result, PGJDBC is able to use streaming behavior to return pagination-related results.
+- **Vectorized Execution**: Queries are now automatically faster for users. This work expands upon the SQL vectorized execution prototype, and allows it to be used in production. Note, this does not cover the long-tail of features necessary to use vectorization on all queries. Vectorized execution is turned on by default for a subset of queries and default back to DistSQL execution for unsupported queries
+- **Improve Streaming Behavior on JDBC Driver**: PostgreSQL wire protocol cursors have been implemented. As a result, PGJDBC is able to use streaming behavior to return pagination-related results. Limitations still apply. For more information, see [Build a Java App with CockroachDB](../v19.2/build-a-java-app-with-cockroachdb.html#recommended-practices).
```

Comment:
Nice! I just merged #5544, so we can update the link to point to `build-a-java-app-with-cockroachdb.html#retrieve-large-data-sets-in-chunks-using-cursors`